Terminal.Gui/View/Layout/DimAuto.cs

```diff
@@ -143,38 +144,42 @@ internal override int Calculate (int location, int superviewContentSize, View us
                 subviewsSize += maxAnchorEnd;
                 #endregion Anchored
 
-                #region Center
-                // Now, handle subviews that are Centered
-                if (dimension == Dimension.Width)
-                {
-                    subviews = us.Subviews.Where (v => v.X is PosCenter).ToList ();
-                }
-                else
-                {
-                    subviews = us.Subviews.Where (v => v.Y is PosCenter).ToList ();
-                }
-
-                int maxCenter = 0;
-                for (var i = 0; i < subviews.Count; i++)
-                {
-                    View v = subviews [i];
-                    maxCenter = dimension == Dimension.Width ? v.Frame.Width : v.Frame.Height;
-                }
-
-                subviewsSize += maxCenter;
-                #endregion Center
+                //#region Center
```

Comment:
This `#region` is pretty unnecessary and just makes the code not sort as expected.